src/corefx/System.Globalization.Native/calendarData.cpp

```diff
@@ -122,24 +123,27 @@ const char* GetCalendarName(CalendarId calendarId)
 		case JAPANESELUNISOLAR:
 		case TAIWANLUNISOLAR:
 			return "chinese";
+		case KOREA:
+			return "dangi";
```

Comment:
can we define such strings as a constants? it is used in multiple place and we need to make sure we'll not misspell any of them